server/pbenchinacan/pbench-server-only.yml

```diff
@@ -0,0 +1,83 @@
+#
+# This configuration is intended to create a brand new (empty) Pbench Server
+# deployment.  The resulting pod consists of three containers: the Pbench Server
+# and its associated Elasticsearch and PostgreSQL database servers.
+#
+# The Elasticsearch and PostgreSQL container images have specific versions and
+# are pulled from cached images in an internal registry; the Pbench Server
+# container image is pulled from the same registry with a configurable tag.
+#
+# The pod should be invoked via the following command
+#
+#    envsubst < server/pbenchinacan/pod-nodata.yml | podman play kube -
+#
+# with the following environment variables defined:
+#
+#  PB_POD_NAME - the name to be used for the pod
+#  PB_INTERNAL_CONTAINER_REG - the FQDN of the container registry
+#  PB_SERVER_IMAGE - the <registry>/<repo>/<img_name> of the Pbench Server image
+#  PB_SERVER_IMAGE_TAG - the tag for the Pbench Server image
```

Comment:
Need to add `PB_DASHBOARD_DIR`, `PB_SERVER_CONFIG`, and `PB_SRV_PBENCH` and remove `PB_INTERNAL_CONTAINER_REG`.